### DISCUSSION TITLE
Disabled icon-only button tooltip

### DISCUSSION THREAD

**Discussion by devlzcode** [adobe/react-spectrum#7038] [Q&A] [answered]
Currently I have a set of buttons that are icon only buttons. One or more can be disabled but because of this the tooltip doesn't work as well. Is there any recommended solutions other than creating a wrapper that uses `useFocusable` and `useObjectRef`? Ideally an option to allow focus/hover states when disabled on a button. On that last note I did notice while looking through the source code that `allowFocusWhenDisabled` is an undocumented prop except it doesn't work to my surprise since I don't see it being omitted anywhere otherwise it technically should work.

Example of my usage:
```ts
<TooltipTrigger>
 <Button iconOnly isDisabled>
   <PlusIcon />
  </Button>
  <Tooltip>...</Tooltip>
 </TooltipTrigger>
 ```

**Comment by snowystinger** [comment, marked as answer]
Please see https://react-spectrum.adobe.com/react-spectrum/Tooltip.html#usage-on-disabled-or-non-interactive-elements
If you're using RAC, we still recommend this pattern. Otherwise, many users will be unable to see your important information. As a sidenote, Tooltips shouldn't be used for anything which conveys critical information, they are already pretty limited accessibility-wise.
